### PR TITLE
LPS-169938 buildLang

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ContributedFragmentEntryDropdownPropsTransformer.js [hunk 1]
```diff
@@ -12,38 +12,61 @@
...
+import CopyFragmentModal from './CopyFragmentModal';
 
-					submitForm(
+const ACTIONS = {
+	copyContributedEntryToFragmentCollection(
+		itemData,
+		portletNamespace,
+		fragmentCollections
+	) {
+		if (!Liferay.FeatureFlags['LPS-166203']) {
```
Comment (sandrodw3) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ContributedFragmentEntryDropdownPropsTransformer.js:26]: We should invert this condition, right? 🤔

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentEntryDropdownPropsTransformer.js [hunk 2]
```diff
@@ -42,29 +44,44 @@ const ACTIONS = {
...
-					const form = document.getElementById(
-						`${portletNamespace}fragmentEntryFm`
-					);
-
-					if (form) {
-						setFormValues(form, {
-							fragmentCollectionId: selectedItem.id,
-							fragmentEntryIds: fragmentEntryId,
-						});
+		if (!Liferay.FeatureFlags['LPS-166203']) {
```
Comment (sandrodw3) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentEntryDropdownPropsTransformer.js:50]: And this one 😄

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/CopyFragmentModal.js [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+function FragmentSetForm({
+	addFragmentCollectionURL,
+	copyFragments,
+	formId,
+	portletNamespace,
+	showNoFragmentCollectionMessage,
+}) {
+	const [name, setName] = useState(
+		showNoFragmentCollectionMessage
+			? Liferay.Language.get('untitled-set')
```
Comment (sandrodw3) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/CopyFragmentModal.js:201]: Um, is not a bit weird giving this title by default? 🤔 what about the typical pattern of leaving it empty and automatically focus the input? I saw the figma has default title and description, but it looks weird to me.

**Review by sandrodw3 — COMMENTED** [review]
This is super beautiful 😍 I left some tiny comments and a little suggestion.

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/CopyFragmentModal.js (hunk 3 above)
Comment (victorg1991) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/CopyFragmentModal.js:201]: Mmm I assume in this case it is about making this easy for the user to not worry about the name. But I'll ask in the channel :)

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ContributedFragmentEntryDropdownPropsTransformer.js (hunk 1 above)
Comment (victorg1991) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ContributedFragmentEntryDropdownPropsTransformer.js:26]: Good catch!

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/CopyFragmentModal.js (hunk 3 above)
Comment (ruben-pulido) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/CopyFragmentModal.js:201]: I had the same question as Sandro :)

**Comment by ruben-pulido** [comment]
ci:reevaluate:1377401_6696

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:reevaluate:1317401_6240

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/127281

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**